crates/sui/tests/checkpoints_tests.rs

```diff
@@ -80,3 +86,69 @@ async fn sequence_fragments() {
         sleep(Duration::from_millis(10)).await;
     }
 }
+
+#[tokio::test]
+async fn end_to_end() {
```

Comment:
That is the new test. Is it the end to end test you were expecting to see (or did I miss something)?

Reply:
@gdanezis

Reply:
good points